x-pack/legacy/plugins/siem/public/containers/case/configure/use_configure.tsx

```diff
@@ -24,8 +25,8 @@ export interface ReturnUseCaseConfigure {
 }
 
 interface UseCaseConfigure {
-  setConnectorId: (newConnectorId: string) => void;
-  setClosureType: (newClosureType: ClosureType) => void;
+  setConnectorId: (newConnectorId: string, newConnectorName?: string) => void;
```

Comment:
Should we change the name from `setConnectorId ` -> `setConnector` ? Also the `setConnectorId` function could take an object `{ newConnectorId, newConnectorName }`.

Reply:
I changed to `setConnector` but I kept the two parameters in the function. I do not think that it needs to be an object yet.